src/web/app/components/sortable-table/sortable-table.component.ts

```diff
@@ -94,4 +98,19 @@ export class SortableTableComponent implements OnInit {
     });
   }
 
+  /**
+   * Sorts the table with initial column index or leftmost sortable column
```

Comment:
Is it possible to do this more explicitly?

Reply:
What do you mean by doing it more explicitly? Like the parent component has to call the method?

Reply:
Use `SortBy.*` instead of `initial column index or leftmost sortable column`.

Reply:
See if you reuse/remove the sorting logic in the CSV generation code (as the sortable table is related to question stats and CSV has a deterministic order for question stats).

Reply:
> resuse/remove the sorting logic in the csv generation code

Will look into this

Reply:
The only way i can see resuse is to add some sort method to the `*-calculation` class. But it seems to be break the purpose of the class since it is supposed to be for calculation of statistics only. Don't think that it brings about much benefit also since they have very different logic(the one in CSV and the one in sortable table). Unless I'm missing something?